ner_v2/detectors/numeral/constant.py

```diff
@@ -0,0 +1,52 @@
+# Number detector constants
+
+NUMBER_DIGIT_UNITS = [0, 1, 1.5, 2, 2.5, 3, 4, 5, 6, 7, 8, 9]
+DIGIT_UNITS = ["zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"]
+NUMERIC_VARIANTS = DIGIT_UNITS + [
+    "ten", "eleven", "twelve", "thirteen", "fourteen", "fifteen",
+    "sixteen", "seventeen", "eighteen", "nineteen", "and",
+    "twenty", "thirty", "forty", "fifty", "sixty", "seventy", "eighty", "ninety",
+    "hundred", "thousand", "million", "billion", "trillion", "fourty", "ninty"
+]
+NUMBER_DATA_FILE_NUMBER = 'number'
+NUMBER_DATA_FILE_NAME_VARIANTS = 'name_variants'
+NUMBER_DATA_FILE_VALUE = 'value'
+NUMBER_DATA_FILE_TYPE = 'type'
+NUMBER_TYPE_UNIT = 'unit'
```

Comment:
'NUMBER_DATA_FILE_VALUE' and 'NUMBER_TYPE_UNIT' have same values as 'NUMBER_DETECT_VALUE' and 'NUMBER_DETECT_UNIT'. It can become confusing. We should refactor this.

Reply:
Refactors all columns :
```# Column of numerals_contant data file fir number detection
NUMBER_DATA_FILE_VARIANTS_COLUMN_NAME = 'name_variants'
NUMBER_DATA_FILE_VALUE_COLUMN = 'value'
NUMBER_DATA_FILE_TYPE_COLUMN = 'type'

# type value of number in numeral_constant data file
NUMBER_TYPE_UNIT = 'unit'

# data file name for number detection
NUMBER_NUMERAL_CONSTANT_FILE = 'numerals_constant.csv'
NUMBER_UNITS_FILE = 'units.csv'

# Column name of units data file for number detection
NUMBER_DATA_FILE_UNIT_VARIANTS_COLUMN = 'unit_variants'
NUMBER_DATA_FILE_UNIT_VALUE_COLUMN = 'unit_value'

# Return type keys of number detection
NUMBER_DETECTION_RETURN_DICT_VALUE = 'value'
NUMBER_DETECTION_RETURN_DICT_UNIT = 'unit'```